pytest_xvfb.py

```diff
@@ -130,8 +131,12 @@ def pytest_addoption(parser):
 
 
 def pytest_configure(config):
-    if config.getoption('--no-xvfb') or not xvfb_available():
+    if config.getoption('--no-xvfb'):
         config.xvfb = None
+    elif not xvfb_available():
+        config.xvfb = None
+        warnings.warn(
+            'Pytest-Xvfb could not find Xvfb installation. Please see that it is installed.')
```

Comment:
I'd say it'd make sense to do so, yeah. For example with qutebrowser, distributions might want to run the tests without X, and that actually works (as the qutebrowser testsuite then simply skips all tests needing a display).